okd/golden-cluster/lb.config.j2

```diff
@@ -53,6 +54,17 @@ backend okd4_k8s_api_be
     server {{ hostvars[be_host].inventory_hostname }} {{ hostvars[be_host].access_ip_v4 }}:6443 check
 {% endfor %}
 
+#!frontend okd4_k8s_api_tls_fe
```

Comment:
maybe a small comment here how this used, as of now it just looks like a commented out section

Reply:
Yes, its commented because the API server certificate won't be available until the cluster is installed and cert-manager configured and certificate issued for API server.

Following steps should be performed before restarting LB (HAProxy LoadBalancer)

1. Copy api.pem (API server certificate) to /etc/haproxy on LB 

    **generate api.pem**
 ```
 oc get secrets user-serving-cert-000 -n openshift-kube-apiserver -o jsonpath='{.data.tls\.crt}'|base64 -d >> api.pem
 oc get secrets user-serving-cert-000 -n openshift-kube-apiserver -o jsonpath='{.data.tls\.key}'|base64 -d >> api.pem
 ```
2. Enable Proxy protocol support in IngressController in openshift-ingress-operator namespace

    `oc edit ingresscontroller/default -n openshift-ingress-operator` 

    Add following to spec
    
    ```
   endpointPublishingStrategy:
      hostNetwork:
        protocol: PROXY
      type: HostNetwork
   ```
3. Add the iptables rule to LB (please change the source ip range according to your network configuration)
    
    `iptables -t nat -A PREROUTING -p tcp ! -s 192.168.0.0/16 --dport 6443 -j REDIRECT --to-port 6444`

4. Remove the comment marker "#!" from /etc/haproxy/haproxy.cfg on LB
5. Restart haproxy service on LB
    
    `sudo service haproxy restart`

Reply:
ok, I can add it to our docs so we can keep track of it there